integration-tests/contracts/ethereum_vrf_contracts.go

```diff
@@ -925,10 +925,10 @@ func (consumer *EthereumVRFBeaconConsumer) RequestRandomnessFulfillment(
 	confirmationDelayArg *big.Int,
 	callbackGasLimit uint32,
 	arguments []byte,
-) error {
+) (*types.Receipt, error) {
```

Comment:
Same question about if a full receipt return is necessary.

Reply:
yes, I need it, since it is used in `RequestAndRedeemRandomness()` and `RequestRandomnessFulfillment()` in order to calculate `requestID`